terasoluna-gfw-dependencies/terasoluna-gfw-jpa-dependencies/pom.xml

```diff
@@ -22,18 +22,32 @@
     </dependency>
     <!-- == End Spring == -->
 
-    <!-- Hibernate -->
+    <!-- == Begin Hibernate == -->
     <dependency>
       <groupId>org.hibernate</groupId>
       <artifactId>hibernate-core</artifactId>
+      <exclusions>
+        <exclusion>
+          <groupId>javax.activation</groupId>
+          <artifactId>javax.activation-api</artifactId>
+        </exclusion>
+      </exclusions>
     </dependency>
+    <!-- == End Hibernate == -->
 
     <!-- == Begin Joda-Time == -->
     <dependency>
       <groupId>org.jadira.usertype</groupId>
       <artifactId>usertype.core</artifactId>
     </dependency>
     <!-- == End Joda-Time == -->
+
+    <!-- == Begin JakartaEE APIs == -->
+    <dependency>
+      <groupId>jakarta.activation</groupId>
+      <artifactId>jakarta.activation-api</artifactId>
+    </dependency>
+    <!-- == End JakartaEE APIs == -->
```

Comment:
- [ ] Please erace `Begin/End JakartaEE APIs`, and move `jakarta.actication-api` into `Begin/End Hibernate`.
You should understand the meaning of the definition in `Begin\End Joda-Time`.

Reply:
fixed the place to define.
9feb6569b9488ae0db3b527db27e81da1dec653e